packages/fxa-payments-server/src/components/PaymentForm/index.test.tsx

```diff
@@ -4,53 +4,11 @@ import '@testing-library/jest-dom/extend-expect';
 import waitForExpect from 'wait-for-expect';
 import { Omit } from '../../lib/types';
 
-// Minimal mock for react-stripe-elements that lets us trigger onChange
-// handlers with testing data
-const MockStripeElement = ({ testid }: { testid: string }) =>
-  class extends React.Component {
-    _ref: null;
-    setRef: (el: any) => void;
-
-    constructor(props: ReactStripeElements.ElementProps) {
-      super(props);
-
-      // Stash this element's onChange handler in module-global registry.
-      const { onChange } = props;
-      onChangeFns[testid] = onChange as onChangeFunctionType;
-
-      // Real react-stripe-elements stash a ref to their container in
-      // this._ref, which we use for tooltip positioning
-      this._ref = null;
-      this.setRef = el => (this._ref = el);
-    }
-
-    render() {
-      return (
-        <div ref={this.setRef} data-testid={testid}>
-          {testid}
-        </div>
-      );
-    }
-  };
+import {
+  mockStripeElementOnChangeFns,
+  elementChangeResponse,
```

Comment:
Refactored this react-stripe-elements mocking stuff into a shared `test-utils` module.